R/htmlwidgets.R

```diff
@@ -269,17 +274,26 @@ shinyRenderWidget <- function(expr, outputFunction, env, quoted) {
         instance$elementId, "\"; Shiny doesn't use them"
       )
     }
-    x <- .subset2(instance, "x")
     deps <- .subset2(instance, "dependencies")
     deps <- lapply(
       htmltools::resolveDependencies(deps),
       shiny::createWebDependency
     )
-    evals = JSEvals(x)
-    list(x = x, evals = evals, deps = deps)
+    payload = c(createPayload(instance), list(deps = deps))
```

Comment:
@yihui That is right. I will leave as-is for now, since it does not create any problems.